scripts/remove-non-beta-product-pages.ts

```diff
@@ -38,9 +38,11 @@ async function main() {
 			dir.name !== '[productSlug]'
 		) {
 			console.log(`🧹 removing pages at /${dir.name}`)
-			await fs.promises.rm(path.join(pagesDir, dir.name), {
-				recursive: true,
-			})
+			if (!process.env.DRY_RUN) {
```

Comment:
Added a `DRY_RUN` env var here so that we can execute the script WITHOUT actually modifying the file system. It allows us to test against the script output.